adapters/Maio/MaioAdapter/GADMAdapterMaioRewardedAd.m

```diff
@@ -13,19 +13,19 @@
 // limitations under the License.
 
 #import "GADMAdapterMaioRewardedAd.h"
-#import <MaioOB/MaioOB-Swift.h>
+#import <Maio/Maio-Swift.h>
 #import <stdatomic.h>
-#import "GADMAdapterMaioAdsManager.h"
 #import "GADMAdapterMaioUtils.h"
 #import "GADMMaioConstants.h"
 
-@interface GADMAdapterMaioRewardedAd () <MaioDelegate>
+@interface GADMAdapterMaioRewardedAd () <MaioRewardedLoadCallback, MaioRewardedShowCallback>
 
 @property(nonatomic, copy) GADMediationRewardedLoadCompletionHandler completionHandler;
 @property(nonatomic, weak) id<GADMediationRewardedAdEventDelegate> adEventDelegate;
-@property(nonatomic, copy) NSString *mediaId;
 @property(nonatomic, copy) NSString *zoneId;
 
+@property(nonatomic) MaioRewarded *rewarded;
```

Comment:
Can you rename this to: `maioRewarded`?

Reply:
Yes. It's possible, but I'd like to see the reason for the change.
rewarded-ad can be uniquely identified under `GADMAdapterMaioRewardedAd`, but what is the motivation for requiring the `maio` prefix? We didn't find such a prefix on any other adapter either.